### ISSUE TITLE
Migration Error After Upgrading to django-mfa2 2.0.0

### ISSUE THREAD

**Issue by swainn** [mkalioby/django-mfa2#23]
Encountered the error after upgrading to 2.0.0 and running database migrations on a fresh database:

```
Applying mfa.0009_user_keys_owned_by_enterprise...Traceback (most recent call last):
  File "/home/nswain/miniconda/envs/tethys-dev/lib/python3.7/site-packages/django/db/backends/utils.py", line 82, in _execute
    return self.cursor.execute(sql)
psycopg2.errors.DatatypeMismatch: column "owned_by_enterprise" is of type boolean but expression is of type integer
LINE 1: update mfa_user_keys set owned_by_enterprise = 0 where key_t...
                                                       ^
HINT:  You will need to rewrite or cast the expression.
```

Looks like an issue with migration `0009_user_keys_owned_by_enterprise.py`.

**Comment by swainn** [comment]
I should add that I was using a PostgreSQL database.

**Comment by mkalioby** [comment]
Fixed in #24
